site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -3,30 +3,38 @@ layout: "layouts/doc-post.njk"
 title: "Declare permissions"
 seoTitle: "Chrome Extensions Declare permissions"
 date: 2012-10-08
-updated: 2014-05-21
+updated: 2023-01-31
 description: An overview of the valid values for the permissions property in manifest.json.
 ---
 
-To use most chrome.\* APIs, your extension must declare its intent in the permissions fields
-of the [manifest][1]. Extensions can request three categories of permissions, specified using the
+To use most [Chrome APIs][api-ref], your extension must declare its intent in the permissions fields
+of the [manifest][doc-manifest]. Extensions can request four categories of permissions, specified using the
 respective keys in the manifest:
 
-* **`permissions`** contain items from a list of known strings (such as "geolocation")
-* **`optional_permissions`** are like regular `permissions`, but are granted by the extension's user at runtime, rather than in advance
-* **`host_permissions`** contain one or more [match patterns][2] that give access to one or more hosts
+`permissions`
+: contain items from a list of known strings (such as `"geolocation"`).
```

Comment:
```suggestion
`permissions`
: Contains items from a list of known strings (such as `"geolocation"`).
```

Reply:
I think I said somewhere that these can almost be thought of as parts of the same sentence. I should have caveated that with, 'don't take that too literally.'

The subject/verb agreement is tricky here because the subject is morphologically plural. However, 'permissions' refers to a single structure making this semantically singular.